backend/databasemodel/alembic/versions/ca6838d8e8a5/upgrade.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE lipas.laavu_kota_kammi RENAME TO laavu_kota_tai_kammi;
+ALTER TABLE lipas.frisbeegolf_rata RENAME TO frisbeegolfrata;
+ALTER TABLE lipas.kavely_ulkoilureitti RENAME TO kavelyreitti_ulkoilureitti;
+ALTER TABLE lipas.laavu_kota_tai_kammi DROP CONSTRAINT laavu_kota_kammi_pk;
+ALTER TABLE lipas.frisbeegolfrata DROP CONSTRAINT frisbeegolf_rata_pk;
+ALTER TABLE lipas.kavelyreitti_ulkoilureitti DROP CONSTRAINT kavely_ulkoilureitti_pk;
```

Comment:
Again, not sure, if this is relevant, but should we do here same thing with constraints as yesterday?

In other words should we have e.g.
ALTER TABLE lipas.kavelyreitti_ulkoilureitti DROP CONSTRAINT IF EXISTS kavelyreitti_ulkoilureitti_pk;
ALTER TABLE lipas.kavelyreitti_ulkoilureitti DROP CONSTRAINT IF EXISTS kavely_ulkoilureitti_pk;
or can we be sure that the constraint name here cannot be kavelyreitti_ulkoilureitti_pk (even if we would have created the database from model.sql)?

Reply:
Yes we are sure here, because the previous migration already changed all the constraints to have the right name :)

Reply:
Or did I forget them from the previous migration, will have to check 👀

Reply:
No, I checked. They are always kavely_ulkoilureitti_pk both in models.sql and the initial migration. Other migrations don't change that name, so we are safe.

Reply:
Great, thanks!